pkg/util/test/util.go

```diff
@@ -3255,8 +3265,8 @@ func ValidateTelemetryV2Disabled(cluster *corev1.StorageCluster, timeout, interv
 			return nil, true, fmt.Errorf("found px-telemetry-phonehome daemonset, waiting for deletion")
 		}
 
-		if validateTelemetryHealthyInPxctl(cluster.Namespace) {
-			return nil, true, fmt.Errorf("failed to validate that Telemetry is Disabled in pxctl status")
+		if err := validateTelemetryStatusInPxctl(false, cluster.Namespace); err != nil {
```

Comment:
same here move it out of the loop and put in the end of validation.

Reply:
Done